src/acme/pd/Courier.java

```diff
@@ -27,14 +30,18 @@ public void setCourierNumber(int courierNumber) {
     }
 
     public static int getNextCourierNumber() {
-		/*try {
-			//FIXME
-			Courier highest = PersistableEntity.querySingle(Courier.class, "select c from COURIER c ORDER BY NUMBER DESC", Collections.EMPTY_MAP);
-			return highest.getCourierNumber() + 1;
+		try {		    
```

Comment:
I know it isn't the cleanest, but it gives us the auto-increment that we weren't getting before.